### DISCUSSION TITLE
BMI stands for Body "What" Index?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#671] [Galaxy Trivia] [answered]
**Bonus:** How to calculate the index?

**Comment by peterbaccaro** [comment, marked as answer]
BMI stands for "**Body Mass Index**". It is a commonly used measurement that estimates the amount of body fat a person has based on their height and weight. The formula to calculate BMI is **weight (in kilograms) divided by height (in meters) squared**. The resulting number is then compared to standardized ranges to determine whether a person is underweight, normal weight, overweight, or obese.

**Comment by SANKARAMDAS** [comment]
BMI- Body Mass Index
